packages/katello/katello-repos/katello-repos.spec

```diff
@@ -5,10 +5,10 @@
 
 %global prereleasesource nightly
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 2
+%global release 3
```

Comment:
You should bump this back to 1.

Background: in RPMs a package build is identified by NEVR: Name, Epoch, Version, Release. Name should be pretty clear, but the others are less obvious. Version is the upstream version. Then the release is for changes in packaging.

Since you're bumping the version, you can reset the release back to 1: the version + release is still newer than before.